cfgov/unprocessed/js/modules/util/breakpoint-state.js

```diff
@@ -26,11 +25,11 @@ function _inBreakpointRange( breakpointRange, width ) {
  *   isBpXS, isBpSM, isBpMED, isBpLG, isBpXL properties.
  */
 function get( width ) {
-  var breakpointState = {};
-  var breakpointKey;
+  const breakpointState = {};
+  let breakpointKey;
   width = width || _getViewportDimensions().width;
 
-  for ( var rangeKey in _breakpointsConfig ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
+  for ( const rangeKey in _breakpointsConfig ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
```

Comment:
Same comment about not needing the inline or max-len ignores if this is moved up a line